pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -65,6 +66,7 @@
 
 @Slf4j
 public class PulsarKafkaConsumer<K, V> implements Consumer<K, V>, MessageListener<byte[]> {
+    private static enum OffsetResetStrategy {EARLIEST, LATEST, NONE}
```

Comment:
Why not using directly `SubscriptionInitialPosition`?

Reply:
Oh, never thought about that. I would take a look. :)

Reply:
Well, I used SubscriptionInitialPosition later. We would still need to seek to the start or the end depending on the config setting. @merlimat Do you want me to get rid of the enum?